src/contents.js

```diff
@@ -676,6 +676,31 @@ class Contents {
 		}.bind(this));
 	}
 
+	_getStylesheetNode() {
+		var styleEl;
+		var key = "epubjs-inserted-css";
+		
+		// Check if link already exists
+		styleEl = this.document.getElementById("#"+key);
+		if (!styleEl) {
+			styleEl = this.document.createElement("style");
+			styleEl.id = key;
+			// Append style element to head
+			this.document.head.appendChild(styleEl);
```

Comment:
This is a behavior change: I think appending the Element is only needed once, right?